### ISSUE TITLE
Offline when launch bc Xbox cloud is blocked at school

### ISSUE THREAD

**Issue by D0D4** [leem919/xcloud-pwa#4]

**Comment by leem919** [comment]
If going to https://xbox.com/play doesn't work on the network, then I can't change anything to fix that since they're responsible for blocking the connection.

**Comment by D0D4** [comment]
> If going to https://xbox.com/play doesn't work on the network, then I can't change anything to fix that since they're responsible for blocking the connection.

It’s not actually blocked its just that its offline on launch i tried on my phone too with my hotspot its just offline with the Xbox logo

**Comment by leem919** [comment]
Do you have access to https://www.xbox.com/favicon.ico ?

This is how I test the internet connection.

**Comment by D0D4** [comment]
> Do you have access to https://www.xbox.com/favicon.ico ?
> 
> This is how I test the internet connection.

Yes i will supply an image of what i see when i open the web app
![IMG_4840](https://github.com/leem919/xcloud-pwa/assets/159467286/3fddaf04-a483-4583-8332-4b079e7d0910)

**Comment by leem919** [comment]
I can't seem to reproduce this error, as I have access to the favicon and the app loads for me. Does it work when you're not on a school network?
